start-mock-server.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
i prefer if we do not run stuff in the background. Easier to make sure it's going to work in all OS. The user can just start the server manually, and the press cmd+c to stop it.

Reply:
ok